anx/provider/loadbalancer/address/address.go

```diff
@@ -69,21 +70,25 @@ type mgr struct {
 	// autodiscover is enabled when prefixAutodiscover is not nil
 	prefixAutodiscover *string
 
+	vipAutoDiscover string
+
 	// statically configured prefixes
 	fixedPrefixes []*prefix
 
 	prefixCache          []*prefix
 	prefixCacheTimestamp time.Time
 }
 
-func newMgr(ctx context.Context, apiClient api.API, legacyClient client.Client) *mgr {
+func newMgr(ctx context.Context, apiClient api.API, legacyClient client.Client, clusterName string) *mgr {
 	m := mgr{
 		api:    apiClient,
 		logger: logr.FromContextOrDiscard(ctx),
 	}
 
 	m.ipam = ipam.NewAPI(legacyClient)
 
+	m.vipAutoDiscover = fmt.Sprintf("kubernetes-lb-vip-%s", clusterName)
```

Comment:
see comment in `NewPrefixWithAutodiscovery`, but at least move this there.

Reply:
Moved it.